src/main/java/com/example/planit/utill/Constants.java

```diff
@@ -10,7 +10,7 @@ public class Constants {
 
     public static final long MINUTES_TO_MILLIS = 60000;
 
-    public static final int HOUR_IN_MINUTES = 60;
+    public static final int MINUTE_IN_SECONEDS = 60;
 
     public static final int SPACE_LIMIT_BETWEEN_EVENTS = 60;
```

Comment:
If you can add the measure unit it would be nice.
`SPACE_LIMIT_BETWEEN_EVENTS` in seconds? minutes?